<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);

</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,15,0,12,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="titleBarFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(41, 44, 68);</string>
        </property>
        <layout class="QHBoxLayout" name="Top">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="TitleBar">
           <property name="styleSheet">
            <string notr="true">color:white;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="minimizeButton">
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>MS Shell Dlg 2</family>
               </font>
              </property>
              <property name="toolTip">
               <string>Minimize</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="maximizeButton">
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Maximize</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="minimumSize">
               <size>
                <width>15</width>
                <height>15</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>17</width>
                <height>17</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Close</string>
              </property>
              <property name="whatsThis">
               <string>Close</string>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border:none;
	border-radius:8px;
	background-color: rgb(174, 84, 252);
}
QPushButton:hover
{
	background-color: rgba(174, 84, 252,150);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="NameImageBar" stretch="1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="BirdNameContainer" stretch="2,3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QWidget" name="EmptyWidget" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(55, 59, 90);</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="MainBirdName" stretch="3,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="BirdName">
              <property name="font">
               <font>
                <pointsize>25</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(55, 59, 90);
color:white;</string>
              </property>
              <property name="text">
               <string>Common Yellowthroat </string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="BinomialName">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(55, 59, 90);
color:white;
border:none;
padding:10px;
padding-top:0px;</string>
              </property>
              <property name="text">
               <string>Geothlypis trichas</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="ImageContainer">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="BirdImageMain">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(55, 59, 90);</string>
            </property>
            <property name="text">
             <string>Bird Image goes here</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="styleSheet">
         <string notr="true">QWidget{
color:white;
padding:10px;

}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="DetailsBar">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="SpeciesContainer">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="SpeciesHeader">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(248,72,193), stop:1 rgb(235,74,203));</string>
               </property>
               <property name="text">
                <string>Species</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="SpeciesValue">
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(248,72,193), stop:1 rgb(235,74,203));
</string>
               </property>
               <property name="text">
                <string>SpeciesValue</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="StatusContainer">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="StatusHeader">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(228,75,209), stop:1 rgb(215,77,219));
</string>
               </property>
               <property name="text">
                <string>Status</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="StatusValue">
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(228,75,209), stop:1 rgb(215,77,219));
</string>
               </property>
               <property name="text">
                <string>StatusValue</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="OrderContainer">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="OrderHeader">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(209,78,224), stop:1 rgb(196,81,234));
</string>
               </property>
               <property name="text">
                <string>Order</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="OrderValue">
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(209,78,224), stop:1 rgb(196,81,234));
</string>
               </property>
               <property name="text">
                <string>OrderValue</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="FamilyContainer">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="FamilyHeader">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(190,82,240), stop:1 rgb(183,83,245));</string>
               </property>
               <property name="text">
                <string>Family</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="FamilyValue">
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(190,82,240), stop:1 rgb(183,83,245));</string>
               </property>
               <property name="text">
                <string>FamilyValue</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="GenusContainer">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="GenusHeader">
               <property name="font">
                <font>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(177,84,250), stop:1 rgb(170,85,255));
</string>
               </property>
               <property name="text">
                <string>Genus</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="GenusValue">
               <property name="styleSheet">
                <string notr="true">border-radius:0;
background-color: qlineargradient(spread:pad, x1:0, y1:0.483, x2:1, y2:0.494, stop:0 rgb(177,84,250), stop:1 rgb(170,85,255));
</string>
               </property>
               <property name="text">
                <string>GenusValue</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DescriptionMap" stretch="2,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="DescriptionContainer" stretch="5,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="Description">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border:None;</string>
            </property>
            <property name="text">
             <string>The common yellowthroat (Geothlypis trichas) is a New World warbler. In the U.S. Midwest, it is also known as the yellow bandit. It is an abundant breeder in North America, ranging from southern Canada to central Mexico. The genus name Geothlypis is from Ancient Greek geo, &quot;ground&quot;, and thlupis, an unidentified small bird; thlypis is often used in the scientific names of New World warblers. The specific  trichas is also from Greek; trikhas is a kind of thrush, the word being derived from trikhos, &quot;hair\</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="margin">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="ButtonLayoutContainer" stretch="2,1,2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="EmptyButtonContainer"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="ReadMoreContainer">
              <item>
               <widget class="QPushButton" name="ReadMoreButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>151</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="baseSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(55, 59, 90);
color:white;
border-radius:5px;</string>
                </property>
                <property name="text">
                 <string>Read More</string>
                </property>
                <property name="iconSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="EmptyButtonContainer1"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="MapContainer" stretch="1,5">
          <item>
           <widget class="QLabel" name="MapLabel">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-left:1px solid black;</string>
            </property>
            <property name="text">
             <string>Map</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Map">
            <property name="styleSheet">
             <string notr="true">border-left:1px solid black;</string>
            </property>
            <property name="text">
             <string>MapGoesHere</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="Footer">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">color: rgb(113, 113, 113);
background-color:rgb(40, 46, 53)</string>
          </property>
          <property name="text">
           <string>Created by Abhishek and Aayush</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
